src/stylesheets/main.css

```diff
@@ -356,24 +345,24 @@ p.important {
     margin-bottom: 5em;
   }
   #details #enabling-contactless-payment {
-    background-color: #c64e5a;
+    background-color: var(--calitp-red-4);
     background-image: initial;
   }
   #details #automating-customer-discounts {
-    background-color: #046b99;
+    background-color: var(--calitp-primary-blue);
     background-image: initial;
     margin-bottom: 0;
   }
   #details #standardizing-trip-quality {
-    background-color: #5b559c;
+    background-color: var(--calitp-purple-4);
     background-image: initial;
   }
   #lastminute #connect {
-    background-color: #c64e5a;
+    background-color: var(--calitp-slate-5);

```

Comment:
Noticed that this only changes the color for Mobile. Then realized that in Desktop, their background is driven by the SVG images and that #152 does the refactor to use CSS-styled divs.